c-dependencies/js-compute-runtime/Makefile

```diff
@@ -41,8 +41,7 @@ CXX_FLAGS += --sysroot=$(WASI_SDK)/share/wasi-sysroot
 
 CFLAGS := -Wall -Werror -Wno-unknown-attributes -Wno-pointer-to-int-cast -Wno-int-to-pointer-cast
 
-LD_FLAGS := -Wl,-z,noexecstack -Wl,-z,text -Wl,-z,relro -Wl,-z,nocopyreloc
-LD_FLAGS += -Wl,-z,stack-size=1048576 -Wl,--stack-first -lwasi-emulated-getpid
+LD_FLAGS := -Wl,-z,stack-size=1048576 -Wl,--stack-first -lwasi-emulated-getpid
```

Comment:
These linker flags never really made sense for wasm, and are now warnings with lld-15.0.6.

Reply:
https://reviews.llvm.org/D119888